tests/Feature/FormTest.php

```diff
@@ -83,7 +88,8 @@ public function test_show_form_with_time_to_complete()
 		$this->assertNotNull($response->json()['time_to_complete']);
 
 		//verificando se o calculo está correto
-		$this->assertEquals(200, $response->json()['time_to_complete']);
+		$expectedResult = $totalTime / $totalRespondents;
+		$this->assertEquals($expectedResult, $response->json()['time_to_complete']);
```

Comment:
```suggestion
		$this->assertEquals($expectedResult, $response->time_to_complete);
```

Acredito que tu pode acessar diretamente esse dado.